src/css/index.css

```diff
@@ -0,0 +1,104 @@
+/* TODO delete this probably
+:root{
+    --title-margin: 20;
+}*/
+
+* {
```

Comment:
this works but better to wrap it in an identifier

Reply:
Just for clarification, you are talking about the '*' which changes the font-family, and not the parameters in the comments I forgot to delete, correct?

Reply:
yeah